app/models/repository.rb

```diff
@@ -23,14 +23,16 @@ class Repository < ApplicationRecord
   class << self
 
     def remove_suse_repos_without_tokens!
-      where(auth_token: nil).where('external_url LIKE ?', 'https://updates.suse.com%').delete_all
+      where(auth_token: nil)
+        .where("external_url LIKE 'https://updates.suse.com%' OR external_url LIKE 'https://dl.suse.com%'")
+        .delete_all
```

Comment:
Let's research a bit why this gets done at all here and if it could possibly get dropped.

Reply:
Removed the function as rmt would not have any repos from updates.suse.com without auth-token

Reply:
done and removed dependency from domain names